data/tabular/blood_brain_barrier_martins_et_al/transform.py

```diff
@@ -0,0 +1,235 @@
+                "description": "corresponding publication",
+            },
+            {
+                "url": "https://rb.gy/0xx91v",
+                "description": "corresponding publication",
+            },
+            {
+                "url": "https://tdcommons.ai/single_pred_tasks/adme/#bbb-blood-brain-barrier-martins-et-al",
+                "description": "data source",
+            },
+        ],
+        "num_points": len(df),  # number of datapoints in this dataset
+        "bibtex": [
+            """@article{Martins2012,
+doi = {10.1021/ci300124c},
+url = {https://doi.org/10.1021/ci300124c},
+year = {2012},
+month = jun,
+publisher = {American Chemical Society (ACS)},
+volume = {52},
+number = {6},
+pages = {1686--1697},
+author = {Ines Filipa Martins and Ana L. Teixeira and Luis Pinheiro
+and Andre O. Falcao},
+title = {A Bayesian Approach to in Silico Blood-Brain Barrier Penetration Modeling},
+journal = {Journal of Chemical Information and Modeling}""",
+            """@article{Wu2018,
+doi = {10.1039/c7sc02664a},
+url = {https://doi.org/10.1039/c7sc02664a},
+year = {2018},
+publisher = {Royal Society of Chemistry (RSC)},
+volume = {9},
+number = {2},
+pages = {513--530},
+author = {Zhenqin Wu and Bharath Ramsundar and Evan~N. Feinberg and Joseph
+Gomes and Caleb Geniesse and Aneesh S. Pappu and Karl Leswing and Vijay Pande},
+title = {MoleculeNet: a benchmark for molecular machine learning},
+journal = {Chemical Science}""",
+        ],
+        "templates": [
+            "The molecule with the {SMILES__description} of {SMILES#} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.",  # noqa: E501
+            "Based on the {SMILES__description} {SMILES#}, the molecule is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.",  # noqa: E501
+            "The {SMILES__description} {SMILES#} represents a molecule that is {penetrate_BBB#not &NULL}identified as {penetrate_BBB__names__adjective}.",  # noqa: E501
+            "The molecule represented with the {SMILES__description} {SMILES#} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.",  # noqa: E501
+            "{SMILES#} represents a molecule that is {penetrate_BBB#not &NULL}identified as {penetrate_BBB__names__adjective}.",  # noqa: E501
+            "{SMILES#} represents a molecule that is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.",  # noqa: E501
+            "{SMILES#} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}.",
+            "The molecule {SMILES#} is {penetrate_BBB__names__adjective}.",
+            # Instruction tuning text templates
+            """Task: Please classify a molecule based on the description.
+Description: A molecule that is {penetrate_BBB__names__adjective}.
+Molecule {SMILES__description}: {SMILES#}
+Constraint: Even if you are uncertain, you must pick either "True" or "False" without using any other words.
+Result: {penetrate_BBB#False&True}""",
```

Comment:
In other places, you used "Answer". Probably not a big deal, but maybe consistency can also be good?

Reply:
Yes, this is something we need to discuss.
I guess we should go for more diversity because otherwise we maybe have to use always a defined prompt.